elephant/spade.py

```diff
@@ -912,6 +913,7 @@ def _fpgrowth(transactions, min_c=2, min_z=2, max_z=None,
                 report='a',
                 algo='s',
                 winlen=winlen,
+                min_neu=min_neu,
```

Comment:
This line appears to be the relevant change if I understand correctly. It seems to address the core issue and provide a solution. Thanks!

Please correct me if I'm wrong about this.